onnxruntime/python/tools/onnx_randomizer.py

```diff
@@ -0,0 +1,87 @@
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation.  All rights reserved.
+# Licensed under the MIT License.
+#--------------------------------------------------------------------------
+
+# An offline standalone script to declassify an ONNX model by randomizing the tensor data in initializers.
+# The ORT Performance may change especially on generative models.
+
+import argparse
+import numpy as np
+from onnx import onnx_pb, numpy_helper, save_model, load_model
+from pathlib import Path
+
+SIZE_THRESHOLD = 10
```

Comment:
where does '10' come from?

Reply:
Just an experimental small value where size of tensor data is bigger than that threshold it can't be shape data.

Reply:
There is TODO below, but it'd be better to put some comments about this in the head for now.

Reply:
commented it